proto/osmosis/poolmanager/v1beta1/query.proto

```diff
@@ -45,8 +59,8 @@ message ParamsResponse { Params params = 1 [ (gogoproto.nullable) = false ]; }
 
 //=============================== EstimateSwapExactAmountIn
 message EstimateSwapExactAmountInRequest {
-  // TODO: CHANGE THIS TO RESERVED IN A PATCH RELEASE
-  string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+  reserved 1;
+  reserved "sender";
```

Comment:
Is this query in the stargate whitelist? If so, then seems important to reason about if this would cause a state break on mainnet. My first pass thought is no, but I don't really know.

Reply:
It is. It's also potentially used by contracts. 

One place that may be state breaking is https://github.com/osmosis-labs/osmosis/pull/4549/files#diff-417b3334c7fd0aefd80aafa70e8feb77a81586dc63b266105aa25bad2ab09a2fL53. It was (unnecessarily) checking that the sender wasn't empty. So a call with an empty sender would've failed before and would pass now.

Reply:
good catch, this leads me to think that we might want to add queries included in stargate queries to e2e testing, since we can mistakenly merge a query PR and have it state breaking